pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -130,7 +140,10 @@ buildStdenv.mkDerivation ({
   inherit src unpackPhase meta;
 
   patches = [
-  ] ++
+  ] ++ lib.optional ((lib.versionAtLeast version "94") && buildStdenv.hostPlatform.isRiscV) (fetchpatch {
```

Comment:
This patch is rather big, but I guess that's ok. Could it be applied unconditionally? Is it (being) upstreamed and do you have a link?
(Just some questions, but always applying is generally preferred if possible.)

Reply:
It could be applied unconditionally, but I think it's not worth the trouble for dealing with possible regressions. The upstream issue is here: https://bugzilla.mozilla.org/show_bug.cgi?id=1717203

Reply:
could you put the link in the nix file?